NEWS

```diff
@@ -1,3 +1,148 @@
+a geodetic object of the database, import and export geodetic objects from/into
+WKT and PROJ strings, and display coordinate operations available between two
+CRSs.
+
+ UPDATES
+ -------
+
+ o Removed projects.h as a public interface (#835)
+
+ o Deprecated the proj_api.h interface. The header file is still available
+   but will be removed with the next major version release of PROJ. It is
+   now required to define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H before the
+   interface can be used (#836)
+
+ o Removed support for the nmake build system (#838)
+
+ o Removed support for the proj_def.dat defaults file (#201)
+
+ o C++11 required for building PROJ (#1203)
+
+ o Added build dependency on SQLite 3.7 (#1175)
+
+ o Added projinfo command line application (#1189)
+
+ o Added many functions to proj.h for handling ISO19111 functionality (#1175)
+
+ o Added C++ API exposing ISO19111 functionality (#1175)
+
+ o Updated cs2cs to use late-binding features (#1182)
+
+ o Removed the nad2bin application. Now available in the proj-datumgrid
+   git repository (#1236)
+
+ o Removed support for Chebyshev polynomials in proj (#1226)
+
+ o Removed proj_geocentric_latitude from proj.h API (#1170)
+
+ o Changed behaviour of proj: Now only allow initialization of
+   projections (#1162)
+
+ o Changed behaviour of tmerc: Now default to the Extended Transverse
+   Mercator algorithm (etmerc). Old implementation available by adding
+   +approx (#404)
+
+ o Chaged behaviour: Default ellipsoid now set to GRS80 (was WGS84) (#1210)
+
+ o Allow multiple directories in PROJ_LIB environment variable (#1281)
+
+ o Added Lambert Conic Conformal (2SP Michigan) projection (#1142)
+
+ o Added Bertin1953 projection (#1133)
+
+ o Added Tobler-Mercator projection (#1153)
+
+ o Added Molodensky-Badekas transform (#1160)
+
+ o Added push and pop coordinate operations (#1250)
+
+ o Removed +t_obs parameter from helmert and deformation (#1264)
+
+ o Added +dt parameter to deformation as replacement for
+   removed +t_obs (#1264)
+
+ BUG FIXES
+ ---------
+
+ o Read +towgs84 values correctly on locales not using dot as comma separator (#1136)
+
+ o Fixed file offset for reading of shift values in NTv1 files (#1144)
+
+ o Avoid problems with PTHREAD_MUTEX_RECURSIVE when using CMake (#1158)
+
+ o Avoid raising errors when setting ellipsoid flattening to zero (#1191)
+
+ o Fixed lower square calculations in rHealpix projection (#1206)
+
+ o Allow Molodensky transform parameters to be zero (#1194)
+
+ o Fixed wrong parameter in ITRF2000 init file (#1240)
+
+ o Fixed use of grid paths including spaces (#1152)
+
+ THANKS TO
+ ------------
+
+ Version 6.0.0 is made possible by the following contributors:
+
+Phil Elson
+Mike Taves
+Mateusz Łoskot
+Markus Neteler
+Jürgen Fischer
+Howard Butler
+Charles Karney
+Alan D.
+Aaron Puchert
+Karoline Skaar
+Marco Bernasocchi
+Mike Toews
+Ben Boeckel
+Philippe Rivière
+Ivan Veselov
+Elliott Sales
```

Comment:
Missing the rest of my name here; I'd guess Alan D. is also truncated.

Reply:
Thanks. Yeah, I need to improve the one-line I use to create this list with - it is not very respectful to people with longer names...

Reply:
shortlog uses tabs, so you can change it to: `awk 'FS="\t" {print $2'}`

Reply:
I did something similar in https://github.com/OSGeo/proj.4/pull/1267/commits/ed4482a87145394b302fce40b060f0d501028c19: `git shortlog -sn 5.2.0..HEAD | tac | awk '{$1=""; print $0}'`